webplugin/js/app/kommunicate.js

```diff
@@ -191,9 +191,15 @@ $applozic.extend(true, Kommunicate, {
         KommunicateUI.showChat();
         KommunicateUI.hideFaq();
     },
-    openConversation: function (groupId) {
+    openConversation: function (groupId, groupDetail) {
         kommunicateCommons.setWidgetStateOpen(true);
-        window.$applozic.fn.applozic('loadGroupTab', groupId);
+        groupDetail &&
+            groupDetail.clientGroupId &&
+            window.$applozic.fn.applozic(
+                'loadGroupTabByClientGroupId',
+                groupDetail
+            );
+        groupId && window.$applozic.fn.applozic('loadGroupTab', groupId);
```

Comment:
calling both is required? loadGroupTabByClientGroupId and loadGroupTab

Reply:
corrected.